flatliners/__init__.py

```diff
@@ -4,3 +4,4 @@
 from .versionedmetrics import VersionedMetrics
 from .clusteralertcorr import ClusterAlertCorrelation
 from .comparisonscore import ComparisonScore
+from .gitversionalertcorr import GitVersionAlertCorrelation
```

Comment:
In a refactoring we should rename that to the same naming scheme as with `std_dev`